### DISCUSSION TITLE
可以支持导出功能吗

### DISCUSSION THREAD

**Discussion by simanya2** [halo-dev/halo#6621] [Q&A]
### 你当前使用的版本

2.18

### 描述一下此特性

可以支持导出pdf、html、markdown、思维导图格式吗

**Comment by ruibaby** [comment]
社区有类似功能的插件：https://github.com/Lyn4ever29/halo-plugin-export-md

目前 Halo 核心暂不考虑支持类似功能，因为这类功能使用插件实现会更加合适。

**Comment by ruibaby** [comment]
现在已经有一个插件支持导出 HTML、Markdown、PDF。

https://github.com/ruibaby/plugin-content-tools/
